gapic-generator-java/pom.xml

```diff
@@ -374,6 +374,7 @@
     <dependency>
       <groupId>com.google.api</groupId>
       <artifactId>gax</artifactId>
+      <type>test-jar</type>
```

Comment:
Can you add source code comment why this test-jar is important?

Reply:
done.